src/app/core/learning-object-module/relevancy/relevancy.service.ts

```diff
@@ -26,13 +26,12 @@ export class RelevancyService {
   /**
    * Sets the nextCheck of a learning object
    *
-   * @param username Username of the author
    * @param learningObjectId learningObjectId
    * @param date The date that nextCheck needs to updated to
    */
-  async setNextCheckDate(username: string, learningObjectId: string, date: Date): Promise<any> {
+  async setNextCheckDate(learningObjectId: string, date: Date): Promise<any> {
```

Comment:
Since `username: string` was removed, you need to update wherever setNextCheckDate is being called. If it's not being called, you need to identify where in the client the the old route implementation was being used, and update the client to use relevancy service.

Reply:
Where`setNextCheckDate` is being called has been updated in this branch, see the `relevancy-date.component.ts` file in the Files changed tab above.